apps/web/src/app/api/uploadthing/core.ts

```diff
@@ -0,0 +1,35 @@
+import { createUploadthing, type FileRouter } from 'uploadthing/next';
+import { UploadThingError } from 'uploadthing/server';
+import { NextRequest } from 'next/server';
+
+const f = createUploadthing();
+
+const getAuthToken = async (req: NextRequest) => {
+  const authToken = req.headers.get('Authorization')?.split(' ')[1] || '';
+  return authToken;
+};
+
+export const ourFileRouter = {
+  imageUploader: f({
+    image: {
+      maxFileSize: '4MB',
+      maxFileCount: 1,
+    },
+  })
+    .middleware(async ({ req }) => {
+      const authToken = await getAuthToken(req);
```

Comment:
The auth token alone gives you nothing without verifying it. You can use the peopleService to get the user. Neon db will do the verification for you.